patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+--- a/src/main/java/net/minecraft/server/TagRegistry.java
++++ b/src/main/java/net/minecraft/server/TagRegistry.java
+@@ -2,7 +2,11 @@ package net.minecraft.server;
+ 
+ import com.google.common.collect.HashMultimap;
+ import com.google.common.collect.Multimap;
++
++import java.util.ArrayList;
++import java.util.List;
+ import java.util.Map;
++import java.util.StringJoiner;
+ import java.util.concurrent.CompletableFuture;
+ import java.util.concurrent.Executor;
+ import java.util.stream.Collectors;
+@@ -82,9 +86,17 @@ public class TagRegistry implements IReloadListener {
+             multimap.putAll("fluids", TagsFluid.b(this.fluidTags));
+             multimap.putAll("entity_types", TagsEntity.b(this.entityTags));
+             if (!multimap.isEmpty()) {
+-                throw new IllegalStateException("Missing required tags: " + (String) multimap.entries().stream().map((entry) -> {
+-                    return (String) entry.getKey() + ":" + entry.getValue();
+-                }).sorted().collect(Collectors.joining(",")));
++                List<String> toSort = new ArrayList<>();
++                for (Map.Entry<String, MinecraftKey> entry : multimap.entries()) {
++                    String s = entry.getKey() + ":" + entry.getValue();
++                    toSort.add(s);
++                }
++                toSort.sort(null);
++                StringJoiner joiner = new StringJoiner(",");
++                for (String s : toSort) {
++                    joiner.add(s);
++                }
++                throw new IllegalStateException("Missing required tags: " + (String) joiner.toString());
+             }
+         }, executor1);
+     }
+diff --git a/src/main/java/net/minecraft/server/TickListChunk.java b/src/main/java/net/minecraft/server/TickListChunk.java
+index 8af1229c3da63a838b0bec1cafde1e41cf5cf551..7ee8534bc656e117fa2a21a39693d53ac8a776b8 100644
+--- a/src/main/java/net/minecraft/server/TickListChunk.java
++++ b/src/main/java/net/minecraft/server/TickListChunk.java
+@@ -84,9 +84,9 @@ public class TickListChunk<T> implements TickList<T> {
+     }
+ 
+     public void a(TickList<T> ticklist) {
+-        this.a.forEach((ticklistchunk_a) -> {
++        for (TickListChunk.a<T> ticklistchunk_a : this.a) {
+             ticklist.a(ticklistchunk_a.a, ticklistchunk_a.d, ticklistchunk_a.b, ticklistchunk_a.c);
+-        });
++        }
+     }
+ 
+     static class a<T> {
+diff --git a/src/main/java/net/minecraft/server/VillagePlaceSection.java b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+index 3aaba7fcaf83a491b9da60d156b7f2379f5a6aeb..5151876951d0089fa2ae229c7fee6e418fea3636 100644
+--- a/src/main/java/net/minecraft/server/VillagePlaceSection.java
++++ b/src/main/java/net/minecraft/server/VillagePlaceSection.java
+@@ -52,7 +52,9 @@ public class VillagePlaceSection {
+         this.c = Maps.newHashMap();
+         this.d = runnable;
+         this.e = flag;
+-        list.forEach(this::a);
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/ArrayList.java#l1226